posthog/api/decide.py

```diff
@@ -64,12 +65,15 @@ def on_permitted_recording_domain(team: Team, request: HttpRequest) -> bool:
     return is_authorized_web_client or is_authorized_mobile_client
 
 
-@csrf_exempt
-@timed("posthog_cloud_decide_endpoint")
-def get_decide(request: HttpRequest):
-    # handle cors request
-    if request.method == "OPTIONS":
-        return cors_response(request, JsonResponse({"status": 1}))
+def get_base_config(token: str, team: Team, request: HttpRequest, skip_db: bool = False) -> dict:
```

Comment:
One thing I'm thinking about as I review this file in general – is it worth get some logs/metrics into this new `get_base_config` path for when we start to migrate over to it down the line?

Reply:
_Potentially_. I'll have a follow up PR about an actual rollout strategy later. Once I have verified the parity of the decide stuff in prod (gonna run tests across all teams to compare) then I'll come up with a slow-and-safe rollout plan